src/mutation/RelayMutation.js

```diff
@@ -230,10 +229,16 @@ class RelayMutation<Tp: Object> {
     return null;
   }
 
-  _resolveProps(props: Tp): void {
-    const fragments = this.constructor.fragments;
-    const initialVariables = this.constructor.initialVariables || {};
+  /**
+   * @internal
+   */
+  static resolveProps(
```

Comment:
What necessitates this change? Mutations will never need a `RelayContext` instance - the changes here seem focused on passing in the instance of `RelayStoreData` instead of using the shared one. 

Let's split the changes to `RelayMutation` and `RelayMutationQueue` into a separate PR.